.pre-commit-config.yaml

```
@@ -27,14 +27,14 @@ repos:
       - id: trailing-whitespace
 
   - repo: https://github.com/pre-commit/mirrors-prettier
```

```suggestion
  - repo: https://github.com/rbubley/mirrors-prettier
```

Avoid alpha releases